{"uuid":"65f21c08-5b49-4acf-9610-2762301986ff","name":"Scenario Outline: The user getting error message with invalid python code for Traversal page","children":["9bea451a-b9f9-4154-8463-0d415f97eef5"],"befores":[{"name":"AppHooks.ApplicationHooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631227959,"stop":1705631227959}],"afters":[{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631228288,"stop":1705631228288},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631228437,"stop":1705631228437},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631229431,"stop":1705631229431},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631229431,"stop":1705631229431},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631230312,"stop":1705631230312},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631230462,"stop":1705631230463},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631230492,"stop":1705631230492},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705631230497,"stop":1705631230497}],"start":1705631227958,"stop":1705631230497}